src/main/resources/org/aerogear/kryptowire/GlobalConfigurationImpl/config.jelly

```diff
@@ -0,0 +1,11 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:section title="Kryptowire">
+        <f:entry title="Kryptowire API key" field="kwApiKey">
+            <f:password />
+        </f:entry>
+        <f:entry title="Kryptowire Endpoint" field="kwEndpoint">
+            <f:textbox />
+        </f:entry>
+    </f:section>
```

Comment:
Not sure if it's just me, but feels like these 2 things should be swapped round so the endpoint field is the first one.